webservices/partition/base.py

```diff
@@ -101,6 +101,7 @@ def run(cls):
             cls.create_child(parent, cycle)
 
         cls.rename()
+        cls.create_trigger()
```

Comment:
Should this method (meaning the `run` method this line is a part of) change more?  If we're now creating the partition in the SQL side, there's no need to create it here.  In other words, by the time this code is run, the master and child tables in the partition should have already been created and setup.  The only thing not done is adding these new triggers, which perhaps we could also shift to the SQL side.

This would mean that other methods in this class, e.g., `create_master`, `create_child`, and `create_indexes`, would go away as well.

What do you think?